tests/lib/rules/no-unused-vars.js

```diff
@@ -40,28 +40,42 @@ tester.run('no-unused-vars', rule, {
     },
     {
       code: '<template scope="props"><span v-if="props"></span></template>'
+    },
+    {
+      code: '<template><div v-for="(item, key) in items" :key="key">{{item.name}}</div></template>'
+    },
+    {
+      code: '<template><div v-for="(v, i, c) in foo">{{c}}</div></template>'
     }
   ],
   invalid: [
     {
       code: '<template><ol v-for="i in 5"><li></li></ol></template>',
-      errors: ['\'i\' is defined but never used.']
+      errors: ["'i' is defined but never used."]
     },
     {
       code: '<template scope="props"></template>',
-      errors: ['\'props\' is defined but never used.']
+      errors: ["'props' is defined but never used."]
     },
     {
       code: '<template v-for="i in 5"><comp v-for="j in 10">{{i}}{{i}}</comp></template>',
-      errors: ['\'j\' is defined but never used.']
+      errors: ["'j' is defined but never used."]
     },
     {
       code: '<template><ol v-for="i in data"><li v-for="f in i"></li></ol></template>',
-      errors: ['\'f\' is defined but never used.']
+      errors: ["'f' is defined but never used."]
     },
     {
       code: '<template><div v-for="(v, i, c) in foo"></div></template>',
-      errors: ['\'v\' is defined but never used.', '\'i\' is defined but never used.', '\'c\' is defined but never used.']
+      errors: ["'v' is defined but never used.", "'i' is defined but never used.", "'c' is defined but never used."]
+    },
+    {
+      code: '<template><div v-for="(v, i, c) in foo">{{i}}</div></template>',
+      errors: ["'v' is defined but never used.", "'c' is defined but never used."]
```

Comment:
You are wrong in the 2nd case.

Reply:
Holly molly! You're absolutely right. Thank you for being so vigilant! I'll update it tomorrow :)